scripts/dtest/docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
       - "0.0.0.0:9000:9000"
     volumes:
       - "./m3dbnode.yml:/etc/m3dbnode/m3dbnode.yml"
+    env:
```

Comment:
@vpranckaitis I don't see where dtests run in CI. figured it was still worth adding to this file as well.

Reply:
👍 for the environment variable. We're not currently running dtest on CI.